C/Coin-OR/Couenne/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+using BinaryBuilder, Pkg
+
+name = "Couenne"
+version = v"0.5.8"
+
+sources = [
+    GitSource("https://github.com/coin-or/Couenne.git",
+              "7154f7a9b3cd84be378d02b483d090b76fc79ce8"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/Couenne
+
+# Remove misleading libtool files
+rm -f ${libdir}/*.la
+update_configure_scripts
+
+# old and custom autoconf
+sed -i s/elf64ppc/elf64lppc/ configure
+
+export CPPFLAGS="${CPPFLAGS} -I${includedir} -I${includedir}/coin -I${includedir}/coin-or"
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+
+if [[ ${target} == *mingw* ]]; then
+    export LDFLAGS="-L${libdir}"
+    export LT_LDFLAGS="-no-undefined"
+fi
+
+./configure \
+    --prefix=$prefix \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --enable-static \
+    --enable-shared \
+    lt_cv_deplibs_check_method=pass_all \
+    --with-asl-lib="-lasl -lipoptamplinterface" \
+    --with-bonmin-lib="-lCoinUtils -lClp -lCgl -lCbc -lbonmin -lbonminampl"
+
+make
+make install
+"""
+
+platforms = supported_platforms()
+platforms = filter!(!Sys.isfreebsd, platforms)
+platforms = expand_cxxstring_abis(platforms)
+platforms = expand_gfortran_versions(platforms)
```

Comment:
Why?  As far as I can see from the log the resulting lib doesn't link to libgfortran.

Reply:
I just started with the same bonmin builder. If it depends on a JLL that expands gfortran, do we not also need to expand?

Reply:
Nope.  Only if the library itself actually links to libgfortran.  Generally speaking, what I do is to not expand anything initially and just look for warnings during audit.

Reply:
Which is also suggested at the end of this section of the manual: https://docs.binarybuilder.org/stable/building/#Platforms